test/system/src/main/java/io/pravega/test/system/framework/services/marathon/PravegaControllerService.java

```diff
@@ -122,18 +121,18 @@ private App createPravegaControllerApp() {
 
         String componentCode = "controller";
 
-        //set env
+        // These properties are defined in "io.pravega.controller.util.Config".
```

Comment:
nit: This comment is unnecessary and slightly problematic: 
* The target class may be refactored with a change in name and namespace, and this comment will then go stale. Ideally, classes should be referenced using a `@link` annotation (when used in JavaDocs) or the like. 
* A simple text search will easily locate the class where these properties are located. 


*